components/python/pyzmq/Makefile

```diff
@@ -10,15 +10,32 @@
 
 #
 # Copyright (c) 2014 Alexander Pyhalov.  All rights reserved.
+# Copyright 2016 Jim Klimov
 #
 
+# See zeromq configure.ac for ABI version mapping to software version
```

Comment:
Of course, you can add LDFLAGS/CFLAGS after include $(WS_MAKE_RULES)/ips.mk.  Perhaps, you have to do LDFLAGS += $(LDFLAGS.$(BITS))